spec/unit/3scale_toolbox_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'fileutils'
```

Comment:
What do you need from FileUtils ? I see only references to File.

Reply:
old stuff, will be deleted